schedule/migration/autoyast_offline_pre@pvm.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: autoyast_offline_pre@pvm
+description: >
+  autoYaST installation and prepare base system for offline migration cases
+vars:
+  AUTOYAST_PREPARE_PROFILE: '1'
```

Comment:
we can remove this one.

Reply:
actually textmode is already set:
offline_sles15sp4_basesys-srv_all_spvm_pre set it in 
https://openqa.suse.de/admin/test_suites so there is no need...

Reply:
Thanks for review, I updated it.